src/components/map/MapService.js

```diff
@@ -779,7 +779,7 @@ goog.require('ga_urlutils_service');
           var dsP = Cesium.KmlDataSource.load(config3d.url, {
             camera: scene.camera,
             canvas: scene.canvas,
-            proxy: new Cesium.DefaultProxy(gaGlobalOptions.ogcproxyUrl)
+            proxy: new Cesium.DefaultProxy(gaGlobalOptions.proxyUrl + 'http/')
```

Comment:
I am a little puzzled with this part of the code. this function is only used in getOlLayerById but we never have kml data sources in our layersConfig. Can you explain when this is used?